Quadriga/src/main/java/edu/asu/spring/quadriga/service/publicwebsite/impl/ProjectStats.java

```diff
@@ -73,4 +97,89 @@ public int compare(IConceptStats o1, IConceptStats o2) {
         });
         return csList;
     }
+
+    private Map<String, IUserStats> getCountofWorkspace(String projectId,
+            Map<String, IUserStats> mapUserWorkspace)
+            throws QuadrigaStorageException {
+
+        List<IProjectCollaborator> projectCollaboratorList = projectCollaboratorManager
+                .getProjectCollaborators(projectId);
+
+        if (projectCollaboratorList == null)
+            return mapUserWorkspace;
+
+        // loop through each collaborator
+        for (IProjectCollaborator projectCollaborator : projectCollaboratorList) {
+            if (projectCollaborator.getCollaborator() != null) {
+                String username = projectCollaborator.getCollaborator()
+                        .getUserObj().getUserName();
+                IUserStats userStats = retrieveUserWorkspaceStats(projectId,
+                        username);
+                mapUserWorkspace.put(username, userStats);
+            }
+        }
+
+        String username = projectDeepMapper.getProjectDetails(projectId)

```

Comment:
Is there a reason you're using the projectdeepmapper here instead of the projectManager?